### ISSUE TITLE
Modify the error obtained for APIs from node-sciduct package from string to object

### ISSUE THREAD

**Issue by murli1201** [NSSAC/CINES#238]
Update the error obtained for APIs from node-sciduct package from string to object eg- {error: {code: 400, message: "bad", data: {.....}}}

**Comment by dmachi** [comment]
I forgot to mention this a couple weeks ago, but I have updated this for the file service.  I haven't yet fixed this for the Job Service.  Can you please check and let me know if you are seeing any problems with this one in relation to files?

**Comment by murli1201** [comment]
@dmachi I checked it. The changes are not yet reflected as you need to make these changes in the node-sciduct package. Once I pull the updated package, I may be getting an object instead of a string.

**Comment by dmachi** [comment]
You’re right, I haven’t updated the node-sciduct package yet. I only updated the service. I will get this done soon.
